library/src/main/java/com/heartlandpaymentsystems/library/terminals/c2x/BaseBuilder.java

```diff
@@ -14,6 +16,9 @@ public void execute() throws Exception {
             throw new Exception("Missing configured device");
         }
 
+        //set the value for allowing duplicates
+        LibraryConfigHelper.setAllowDuplicates(allowDuplicates);
```

Comment:
When this was in `C2XDevice`, we were calling `LibraryConfigHelper.setAllowDupes`. Which is the correct method on `LibraryConfigHelper`?